docs/modules/ROOT/pages/shipping-logs-to-amazon-elasticsearch-service.adoc

```diff
@@ -1,71 +1,109 @@
+Create an index pattern to load the logs received from OpenSearch into a Kibana dashboard. 
 
-== Set up a Kibana Dashboard
+You'll need the master user credentials and Kibana URL that you created when you <<configure-opensearch-domain, configured OpenSearch>>.
 
-In Elasticsearch Service instance detail page, you can see the URL for Kibana, please click on it and you will see a login page. Use username/password as you defined in the creation phase.
+[NOTE]
+====
+To create log data for viewing, check for the following:
 
-image:kibana-login.png[A login page for Kibana]
+* Your {hazelcast-cloud} cluster is running 
+* A client is writing data to the cluster.
+====
 
-Once you logged in, you can skip the onboarding part, and click Discover menu on the left-hand-side to define the index pattern. Please put the *wildcard* char in the text box to handle all the logs coming from Hazelcast Cloud Clusters.
+. From your domain details page in OpenSearch, click the Kibana URL.
+. Log in to Kibana, using your master user credentials.
+. Select *Explore on my own* to skip onboarding.
+. From the *Select tenant* dialog box, select *Private* and click *Confirm*.
+. From the main menu, select *Kibana* > *Discover* to start your set up.
+. Click *Create index pattern*. An index pattern tells Kibana which indices contain the data that you want to view.
+. In the *Index pattern* field, enter a wildcard `*` to find all log data received from OpenSearch. 
+. When Kibana finds the {hazelcast-cloud} logs, click *Next step*.
+. From the *Time field* list, select *updated_at* so that you'll be able to filter your log file data by time.
+. Click *Create index pattern*. Your log file data is ready to view. Simply select *Kibana* > *Discover* from the main menu.
 
-image:kibana-index-patterns.png[A * wildcard is displayed in the Index pattern field]
++
+image:kibana-dashboard.png[Log data in Kibana Dashboard]
 
-Once you click the Next step, you can skip the Settings part then you will see some aggregated table data for summary and now you are ready to discover data of Hazelcast Cloud Clusters.
+If necessary, you can now return to your OpenServe domain and increase the number of nodes or availability zones to add resilience to the logging service.
 
-image:kibana-index-pattern-settings.png[A time filter called updated_at is listed at the top of a dropdown]
+== Summary
 
-You can go to the Discover page from the left menu and see logs coming from Hazelcast Cloud Cluster.
+In this tutorial, you learned how to do the following:
 
-image:kibana-hazelcast-cluster-log.png[Hazelcast Cloud cluster logs are displayed in Kibana]
+* Create an Amazon OpenSearch domain to receive log files from your {hazelcast-cloud} cluster.
+* Configure your Dedicated cluster to automatically send log files to the OpenSearch domain.
+* Load log files into a Kibana dashboard using an index pattern.
```

Comment:
Would be good to include next steps or related resources so that users can go somewhere else after the tutorial.